modules/mux/targets/host/source/device.cpp

```diff
@@ -552,6 +554,12 @@ host::arch device_info_s::detectHostArch() {
 #elif defined(UTILS_SYSTEM_64_BIT)
   arch = host::arch::X86_64;
 #endif
+#elif defined(UTILS_SYSTEM_RISCV)
+#if UTILS_SYSTEM_64_BIT
```

Comment:
This could probably use the same style as the rest here (`#ifdef UTILS_SYSTEM_32_BIT`/.../`#elif defined(UTILS_SYSTEM_64_BIT)`/...)?